packages/infra/dashboards/cw-dashboard-prod.json

```diff
@@ -326,8 +326,8 @@
                 "annotations": {
                     "horizontal": [
                         {
-                            "label": "VolumeWriteIOPs >= 5000 for 1 datapoints within 5 minutes",
-                            "value": 5000,
+                            "label": "VolumeWriteIOPs >= 100000 for 1 datapoints within 5 minutes",
+                            "value": 100000,
```

Comment:
This should be 10_000, right?

Reply:
Yes. Updated. Note that JSON does not accept underscores as separators inside numbers